src/components/controls/miscInfoText.js

```diff
@@ -27,11 +27,15 @@ export const AnimationOptionsInfo = (
   </>
 );
 
-export const PanelOptionsInfo = (
+export const PanelLayoutInfo = (
   <>
-    Control which panels are being displayed and whether to show the tree and the map side-by-side (<em>grid</em>) or expanded (<em>full</em>).
-    <br/>
-    Note that what options are available here are dataset specific!
+    Control whether to show the tree and the map side-by-side (<em>grid</em>) or expanded (<em>full</em>).
+  </>
+);
+
+export const EntropyInfo = (
+  <>
+    FIXME: Add some info text that describes the entropy panel.
```

Comment:
Done in [force-push](https://github.com/nextstrain/auspice/compare/b3bceb5a7d2a672bea05e7e4f8320122280ca01c..bf037f1bfc8a318efd68454465843ea910fbf1e6). I copied the text from the [in-panel info hover](https://github.com/nextstrain/auspice/blob/bf037f1bfc8a318efd68454465843ea910fbf1e6/src/components/entropy/index.js#L280).

I'm also realizing now that the other info tooltips were written to describe panel options, not panels themselves. I think, now that each section clearly represents a panel as a whole, it would be good to at least add an introductory line to describe each panel. What do you think?

Reply:
> Done in [force-push](https://github.com/nextstrain/auspice/compare/b3bceb5a7d2a672bea05e7e4f8320122280ca01c..bf037f1bfc8a318efd68454465843ea910fbf1e6). I copied the text from the [in-panel info hover](https://github.com/nextstrain/auspice/blob/bf037f1bfc8a318efd68454465843ea910fbf1e6/src/components/entropy/index.js#L280).

👍 

> now that each section clearly represents a panel as a whole, it would be good to at least add an introductory line to describe each panel.

I agree

Reply:
Done: 830b73de2d4d03f42b7272e8fd6f83e6616cfed1